src/game/engine/bullettype.h

```diff
@@ -71,16 +71,32 @@ class BulletTypeClass : public ObjectTypeClass
     virtual ObjectClass *Create_One_Of(HouseClass *house) const override { return nullptr; }
     virtual BOOL Read_INI(GameINIClass &ini);
 
-    int Get_Type() const { return Type; }
-    BOOL Is_Anti_Air() const { return AntiAir; }
-    BOOL Is_Anti_Ground() const { return AntiGround; }
-    BOOL Is_Inaccurate() const { return Inaccurate; }
     void Code_Pointers() {}
     void Decode_Pointers() {}
 
-    BOOL Get_UnderWater() const { return UnderWater; }
+    BOOL Get_High() const { return m_High; }
+    BOOL Get_Shadow() const { return m_Shadow; }
+    BOOL Get_Arcing() const { return m_Arcing; }
+    BOOL Is_Dropping() const { return m_Dropping; }
+    BOOL Is_Inviso() const { return m_Inviso; }
+    BOOL Get_Proximity() const { return m_Proximity; }
+    BOOL Get_Animates() const { return m_Animates; }
+    BOOL Is_Ranged() const { return m_Ranged; }
+    BOOL Rotates() const { return !m_NoRotate; }
+    BOOL Is_Inaccurate() const { return m_Inaccurate; }
+    BOOL Is_Translucent() const { return m_Translucent; }
+    BOOL Is_Anti_Air() const { return m_AntiAir; }
+    BOOL Is_Anti_Ground() const { return m_AntiGround; }
+    BOOL Is_Anti_Sub_Warfare() const { return m_AntiSubWarfare; }
+    BOOL Get_Degenerates() const { return m_Degenerates; }
+    BOOL Is_UnderWater() const { return m_UnderWater; }
+    BOOL Get_Parachuted() const { return m_Parachuted; }
+    BOOL Get_Gigundo() const { return m_Gigundo; }
+    uint8_t Get_ROT() const { return m_ROT; }
+    int Get_Arm() const { return m_Arm; }
+    int Get_Frames() const { return m_Frames; }
```

Comment:
Bit inconsistent here, not deal breaking and I'll merge as is, but rather than Get_ I think for bools it should either just be Is_ or Has_ where appropriate or where its fairly unambiguous just the word such as Animates rather than Get_Animates. Get_ should be reserved for returning a value.